src/main/java/service/ShopService.java

```diff
@@ -0,0 +1,76 @@
+package service;
+
+import repo.Order.Order;
+import repo.Order.OrderRepo;
+import repo.Product.Product;
+import repo.Product.ProductRepo;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class ShopService {
+
+    public static void main(String[] args) throws IOException {
+
+        Product product1 = new Product(41465, "Cheeseburger");
+        Product product2 = new Product(44414, "Hamburger");
+        Product product3 = new Product(41256, "Pommes");
+        Product product4 = new Product(41255, "Wurst");
+
+
+        ProductRepo productRepo = new ProductRepo();
+
+        productRepo.addProduct(1, product1);
+        productRepo.addProduct(2, product2);
+        productRepo.addProduct(3, product3);
+        productRepo.addProduct(4, product4);
+
+
+        productRepo.listAllProducts();
```

Comment:
Pro-Tipp (in Pseudocode)

Map productList = productRepo.listAllProducts();
sout(productList)